hack/build/releases.md

```diff
@@ -0,0 +1,93 @@
+# Version and Release
+
+* [Overview](#overview)
+* [Version Scheme](#version-scheme)
+* [Releasing a New Version](#releasing-a-new-version)
+    * [Promoting the Release](#promoting-the-release)
+    * [PROW CI](#prow-ci)
+# Overview
+
+## Version Scheme
+
+The **kubevirt-velero-plugin** has similar release process to the kubevirt and cdi. 
+It adheres to the [semantic version definitions](https://semver.org/) format of vMAJOR.MINOR.PATCH.  These are defined as follows:
+
+- Major - Non-backwards compatible, API contract changes.  Incrementing a Major version means the consumer will have to 
+make changes to the way they interact with the Velero Backup/Restore API.  Another option is the need to follow Major version 
+change in the velero project. When these changes occur, the Major version will be incremented at the end of the sprint 
+instead of the Minor Version.
+
+- Minor - End of Sprint release. Encapsulates non-API-breaking changes within the current Major version.  
+The current Sprint cycle is 2 weeks long, producing in bug fixes and feature additions. Publishing a Minor version 
+at the end of the cycle allows consumers to immediately access the end product of that Sprint's goals. Issues or bugs 
+can be reported and addressed in the following Sprint.
+
+- Patch - mid-Sprint release for fixing blocker bugs. In the case that a bug is blocking kubevirt-velero-plugin consumers' workflow, 
+a fix may be released as soon as it is merged.  A Patch should be limited expressly to the bug fix and not include anything unrelated.
+
+## Releasing a New Version
+
+Release branches are used to isolate a stable version of kubevirt-velero-plugin.  Git tags are used within these 
+release branches to track incrementing of Minor and Patch versions.  When a Major version is incremented, a new 
+stable branch should be created corresponding to the release.
+
+- Release branches should adhere to the `release-v#.#.#` pattern.
```

Comment:
release branches are major/minor, so X.Y, not X.Y.Z, Zs are tags.

Reply:
done